docs/getting-started/index.mdx

```diff
@@ -31,8 +31,14 @@ If you want to disable login for a single-user setup, set [`WEBUI_AUTH`](/gettin
 You cannot switch between single-user mode and multi-account mode after this change.
 :::
 
-:::danger
-When using Docker to install Open WebUI, make sure to include the `-v open-webui:/app/backend/data` in your Docker command. This step is crucial as it ensures your database is properly mounted and prevents any loss of data.
+:::info
+This tutorial uses [Docker named volumes](https://docs.docker.com/storage/volumes/) to guarantee the **persistance of your data**. This might make it difficult to know exactly where your data is stored in your machine if this is your first time using Docker. Alternatively, you can replace the volume name with a absolute path on your host machine to link your container data to a folder in your computer using a [bind mount](https://docs.docker.com/storage/bind-mounts/).
```

Comment:
we might want keep the original message and to put these in details so they're hidden by default, the content lgtm!

Reply:
cool! i will do that